egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -4,11 +4,14 @@
 from __future__ import print_function
 import os
 import argparse
+import shlex
 import sys
 import warnings
 import copy
 import imp
 import ast
+import scipy.signal as signal
```

Comment:
I moved scipy and numpy imports to AddLpFilter() please don't move the import statements.